app/lib/email.server.ts

```diff
@@ -42,6 +42,11 @@ function getBaseMessage({
   subject,
   body,
 }: BaseMessageProps): Omit<SendEmailCommandInput, 'Destination'> {
+  if (getHostname().startsWith('dev') || getHostname().startsWith('test')) {
+    body =
+      `******** This notification is from the internal development version of GCN. For the production version, go to https://gcn.nasa.gov/ ********
```

Comment:
We have some code here that generates different dev banner text depending on the hostname here: https://github.com/nasa-gcn/gcn.nasa.gov/blob/main/app/components/DevBanner.tsx#L21-L33

Please refactor to use this code in both places.